src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java

```diff
@@ -692,10 +720,31 @@ private boolean isEmpty(List<?> l) {
         }
 
         @Override
-        public List<CreateTableRequest> getPhysicalTables() {
-            return createTableRequests;
+        public List<CreateTableRequest> getStaticPhysicalTables() {
+            return staticPhysicalTables;
+        }
+
+        @Override
+        public CreateTableRequest getDynamicPhysicalTable(DynamoTableDescription virtualTable) {
+            String physicalTableName = prefix(tablePrefix, virtualTable.getTableName());
```

Comment:
Do we need/want to enforce namespacing between the static physical tables configured on the adapter and the dynamic physical tables created for virtual mt tables? If not, what would happen if we create a virtual table with the same name as a static physical table?

Reply:
Yes I think that's a good idea. Perhaps dynamic physical tables can have a different prefix, which would make isPhysicalTables() more robust / make it less likely that we accidentally pick up other tables like .leases tables.